flake.nix

```diff
@@ -53,19 +56,16 @@
         basePath = toString src + "/";
       in
         pkgs.lib.cleanSourceWith {
-          filter = (
-            path: type: let
-              relPath = pkgs.lib.removePrefix basePath (toString path);
-              includePath =
-                (type == "directory")
-                || pkgs.lib.any
-                (re: builtins.match re relPath != null)
-                regexes;
-            in
-              # uncomment to debug:
-              # builtins.trace "${relPath}: ${lib.boolToString includePath}"
-              includePath
-          );
+          filter = path: type: let
+            relPath = pkgs.lib.removePrefix basePath (toString path);
+            includePath =
+              (type == "directory")
+              || pkgs.lib.any (re: builtins.match re relPath != null)
+              regexes;
+            # uncomment to debug:
+            # builtins.trace "${relPath}: ${lib.boolToString includePath}"
```

Comment:
Moving this into the `let ... in` statement is just plain wrong, no? o.0

Reply:
It seems that if you manually move it back and re-run `nixfmt`, they (both `alejandra` and `nixfmt`) permit/keep it.